src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
Good point. I copied this from the [`Row` component](https://github.com/react-bootstrap/react-bootstrap/blob/1bc32530e1c8cfa861ad7bbfa4bb943a3b0ecd8d/src/Row.js#L5-L7). [`Navbar`](https://github.com/react-bootstrap/react-bootstrap/blob/1bc32530e1c8cfa861ad7bbfa4bb943a3b0ecd8d/src/Navbar.js#L18) uses `node` as well.

The react documentation says the following about the `node` PropType:
> Anything that can be rendered: numbers, strings, elements or an array containing these types.

That said, I think `node` is the correct type, consider the following example:

```js
<Jumbotron componentClass={MyCustomSectionComponent}>
   <strong>foo</strong>
</Jumbotron>
```